docs/topic_guide/index.rst

```diff
@@ -0,0 +1,14 @@
+.. _aiapy-topic-guide-index:
+
+*****************
+aiapy topic guide
```

Comment:
I'm not really sure either of these are topic guides. Maybe the data prep one could be with some tweaking but I don't think it's worth having a whole section of the docs just for a single entry.

Reply:
Sure. I will think again.